src/server/scripts/Northrend/IcecrownCitadel/boss_professor_putricide.cpp

```diff
@@ -1658,6 +1658,27 @@ class spell_stinky_precious_decimate : public SpellScriptLoader
         }
 };
 
+// 70402, 72511, 72512, 72513 - Mutated Transformation
+class spell_abomination_muted_transformation : public SpellScript
+{
+    PrepareSpellScript(spell_abomination_muted_transformation);
+
+    void HandleResistence(SpellEffIndex /*effIndex*/)
+    {
+        Unit* caster = GetCaster();
+        uint32 damage = GetHitDamage();
+        Unit* target = GetHitUnit();
+        damage -= Unit::CalcSpellResistedDamage(caster, target, GetHitDamage(), SPELL_SCHOOL_MASK_SHADOW, nullptr);
```

Comment:
Maybe with only one line of CalcSpellResistedDamage changing SPELL_SCHOOL_MASK... with GetSpellInfo()->GetSchoolMask() can be better?

Reply:
Nope, need be splitted in 2, or it will have same effect of default behavior